### PR TITLE
added button and quantity Counter components

### PR DESCRIPTION
**Pull request by amandhunna**
- added new basehelper functions, 
- added button and quantity Counter components
- added css varialbles

### CODE REVIEW

File: client/src/App.css [hunk 1]
```diff
@@ -1,4 +1,34 @@
...
\ No newline at end of file
+:root {
+  --primary-text: 0.8vw;
+  --primary-color: #5c5c5c;
+
+  --secondary-text: 0.7vw;
+  --secondary-color: #5c5c5c3d;
+
+  --btn-color: #939393;
+  --btn-color1: #ff9999;
```
Comment (gaurav7557) [inline client/src/App.css]: use button color primary and secondary as names

File: client/src/components/Button/button.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
+.button {
+  background: var(--btn-color1);
+  width: 100%;
+  padding: 1rem;
+  text-transform: uppercase;
+  letter-spacing: 2px;
```
Comment (gaurav7557) [inline client/src/components/Button/button.css]: use em

File: client/src/helper/BaseHelper.js [hunk 3]
```diff
@@ -19,10 +19,22 @@ class BaseHelper {
...
+    }
+
     async hashPassword(password) {
         const hash =  crypto.createHash('sha1').update(password, "utf8").digest("hex");;
         return hash;
     }
+
+    getPrice(currencyType, price) {
+        const currencySymbol = {
+            'in': 'Rs.'
```
Comment (gaurav7557) [inline client/src/helper/BaseHelper.js:34]: use rupee symbol

**Review by gaurav7557 — APPROVED** [review]

**Comment by amandhunna** [comment]
Merged with cart page

**Pull request closed without merging**